imports/plugins/core/dashboard/client/components/toolbar.js

```diff
@@ -36,7 +36,7 @@ class PublishControls extends Component {
 
   onViewContextChange = (event, isChecked) => {
     if (typeof this.props.onViewContextChange === "function") {
-      this.props.onViewContextChange(event, isChecked ? "administrator" : "customer");
+      this.props.onViewContextChange(event, isChecked ? "customer" : "administrator");
     }
```

Comment:
And I was wondering where that came from @aaronjudd 
![image](https://cloud.githubusercontent.com/assets/670954/22725287/2e6091c0-edd5-11e6-8985-896635fbbea1.png)